src/app/navbar/navbar.component.css

```diff
@@ -108,6 +108,30 @@
   }
 }
 
+@media screen and (max-width:1020px) {
```

Comment:
you already have a lot media css group them in ascending order and merge some media css which could be merged. in this way it would help us better in future developments.